### ISSUE TITLE
Unable to import sip

### ISSUE THREAD

**Issue by Pathfinder216** [PyCQA/pylint#3889]
I'm trying to create a custom metaclass like in #2025, but the solution provided there isn't working. I believe the root cause is pylint being unable to import sip.

I've tried pipenv, virtualenv, and just pip for installing. I've tried the latest pylint and the pre-release version of pylint. I tried adding the `--unsafe-load-any-extension=y` argument.

### Steps to reproduce

```
import sip
```

### Current behavior

> E0401: Unable to import 'sip' (import-error)

### Expected behavior
No `import-error` is shown

### pylint --version output

> pylint 2.6.0
> astroid 2.4.2
> Python 3.8.5 (tags/v3.8.5:580fbb0, Jul 20 2020, 15:57:54) [MSC v.1924 64 bit (AMD64)]

**Comment by hippo91** [comment]
@Pathfinder216 i think your problem is not related to `pylint`. I installed sip via `pip install sip` but was unable to import it inside a python REPL. In fact the package `sip` seems to give access to `sipbuild` module. Replace `import sip` by `import sipbuild` and `pylint` won't complain anymore.

**Comment by Pathfinder216** [comment]
After looking into it more closely, I discovered the solution. I'm using `PyQt5`, which apparently uses its own version of `sip`. When switching the import statement to `import PyQt5.sip`, it worked as expected. I'm not really sure it didn't crash Python when importing `sip` directly before, but I was able to recreate that issue by deleting and recreating my virtual environment.

Sorry for the frivolous issue, and thanks for looking into it!
